hexa/datasets/schema/mutations.py

```diff
@@ -252,11 +253,19 @@ def resolve_create_version_file(_, info, **kwargs):
                     uri=version.get_full_uri(mutation_input["uri"]),
                     content_type=mutation_input["contentType"],
                 )
-                return {
-                    "success": True,
-                    "errors": [],
-                    "file": file,
+            dataset_file_metadata_queue.enqueue(
```

Comment:
I think we should add a `load_metadata` on the DatasetVersionFile and call that on the save() method if there are no metadata attached.

Reply:
So on creation of DatasetVersionFile you suggest to change the flow to trigger generation of metadata file on save:
- DatasetVersionFile.save() -> load_metadata() -> enqueue(task)
Where load_metadata will enqueue the sample generation task, which does all the job to download, link with FK to DatasetVersionFile and save in db the sample